lib/python/rose/suite_engine_procs/cylc.py

```diff
@@ -717,6 +717,7 @@ def get_suite_jobs_auths(self, suite_name, cycle_time=None,
             stmt = "SELECT DISTINCT user_at_host FROM task_jobs"
             stmt_where_list = []
             stmt_args = []
+            break
```

Comment:
Logic of this block and similar blocks has been moved to a separate method in the latest commit.